### ISSUE TITLE
Field doesnt exist. Not possible to delete

### ISSUE THREAD

**Issue by kurt-hardy** [prisma/migrate#577]
Related - report id 10921

Hi

I created a migration to add "DateCreated" to a User and then up-applied it and deleted it. Now I have no reference to DateCreate on a user and I cant run any command.

User:
```
model User {
  id Int @default(autoincrement()) @id
  email String @unique
  firstName String?
  lastName String?
  isTaxExempt Boolean @default(false)
  renewalDate DateTime @default(now())
  userStatus String @default("public")
}
```

Any ideas?
Thanks

**Comment by janpio** [comment]
> and I cant run any command.

What does that mean? Do you get an error message?

**Comment by Jolg42** [comment]
The error from the error report is 
`Failure during a migration command: Generic error. (error: The field dateCreated on model User does not exist in this Datamodel. It is not possible to delete it.)` from 2.5.1

**Comment by tomhoule** [comment]
This is a case of "migrations folder out of sync with _Migration table". The current accepted solution is to delete both.

**Comment by tomhoule** [comment]
Thanks for taking the time to report this issue!

We've recently released a Prisma Migrate Preview ([2.1.3.0 release notes](https://github.com/prisma/prisma/releases/tag/2.13.0)), which has quite a few changes compared to the previous experimental version. We believe this issue is no longer relevant in this new version, so we are closing this.

We would encourage you to try out Prisma Migrate Preview. If you come across this or any other issue in the preview version, please feel free to open a [new issue in prisma/prisma](https://github.com/prisma/prisma/issues/new/choose).
For general feedback on Prisma Migrate Preview, feel free to chime in [on this issue](https://github.com/prisma/prisma/issues/4531).
